code/modules/research/designs/medical.dm

```diff
@@ -88,11 +88,20 @@
 	build_path = /obj/item/device/healthanalyzer/improved
 	sort_string = "KBAAB"
 
+/datum/design/item/medical/advanced_analyzer
+	name = "advanced health analyzer"
+	desc = "An even more advanced handheld health scanner, complete with a full biosign monitor and on-board radiation and neurological analysis suites."
+	id = "improved_analyzer"
```

Comment:
Same ID as improved_analyzer design.

Reply:
Whoops, copy and paste error. Let me fix that